packages/wallets/src/components/WalletsCarouselContent/WalletsCarouselContent.tsx

```diff
@@ -119,7 +119,7 @@ const WalletsCarouselContent: React.FC<TProps> = ({ onWalletSettled }) => {
         if (activeWallet) {
             setSelectedLoginId(activeWallet?.loginid);
         }
-    }, [activeWallet]);
+    }, [activeWallet?.loginid]);
```

Comment:
```
useEffect(() => {
        if (activeWallet.loginid) {
            setSelectedLoginId(activeWallet.loginid);
        }
    }, [activeWallet?.loginid]);
   ```

Reply:
@wojciech-deriv could we apply this suggestio to avoid githib complaining

Reply:
noted, will updated with next PR as this is ready to be merged,